docs/source/pages/configuration.rst

```diff
@@ -280,6 +280,14 @@ Settings
 | **Default**: ``'refresh_token'``
 |
 
+-----------------
+``cookie_samesite``
+-----------------
+
+| **Purpose**: Adds a ``samesite`` field to cookies. Can be set to ``'Strict'``, ``'Lax'``, ``'None'`` or ``None`` (note the difference in the quotes). This should be used in production, but is disabled by default because it might lead to unintended frustrations in development.
+| **Default**: ``None``
```

Comment:
I am not sure I agree that `None` is the right default.

I believe that most browsers now are defaulting to `Lax`. [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#lax). Without digging deeper, I am pretty sure Chrome is the same.

Reply:
You are right, `Lax` is indeed the standard default value for SameSite.

Reply:
If I understood your logic, the idea was to make it simpler for development. If that's the case, I would be happy to accept `None` as a default when in debug mode. But, honestly, debug mode in Sanic JWT is something that probably needs an overhaul anyway.

Reply:
Yes, that's exactly it. But it is better to leave it `Lax` - even in debug mode! That's because browsers require HTTPS to be used when setting cookies with the `None` flag.

Reply:
In the end, it's up to you to decide. If in `sanic_jwt.configuration` we set `config["cookie_samesite"]: None`, then `sanic-jwt` won't add a SameSite flag at all by default. If it is instead set to `"Lax"`, then every cookie that `sanic-jwt` sets will have `SameSite=Lax` in it.

Reply:
I updated the branch to auto set SameSite to `Lax`, but pytest did complain, so I'll look further into it.

Reply:
I just pushed a fix to the pytest complaint. If you decide to use `None` instead of `"Lax"`, the first commit on this branch is valid.